index.html

```diff
@@ -195,25 +201,26 @@ <h2 class="uppercase text-gray-700 text-sm">commercial examples</h2>
 							}
 						})
 					}
+                    window.scrollTo(0,0);
 				},
 				styleBox(style) {
 					let stat = this.displayBy.index
 					let background = "bg-gray-300"
-
 					for(let i in this.mappings.background) {
 						if(this.mappings[stat][i] >= style.label[stat]) {
 							break
 						}
 						background = "bg-" + this.displayBy.background + "-" + this.mappings.background[i]
 					}
-
 					if(this.selected) {
-						if(this.selected.id === style.id) {
+						if(this.selected.id === style.id) {  
+                             window.scrollTo(0,0); 
```

Comment:
Why are you jumping to top three times in this method?

Reply:
I did this because there is a scroll issue on mobile  when you trigger the rebuild of  grid,  a single one didn't  solve the problem.  I believe its the machine  rebuilding page and  trying to go to end of page? - or at least looks that way I think  it may be a  quirk of  remembering scroll feature, but that seems unlikely, also it doesn't  trigger in all instances of  grid pressing.